### PR TITLE
feat: added jmeterd

### PR DESCRIPTION
**Pull request by ypoplavs**
## Pull request description 
Added jmeterd executor.

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube-api/executors.json [hunk 1]
```diff
@@ -134,6 +134,46 @@
...
+                "<runPath>",
+                "-l",
+                "<jtlFile>",
+                "-e",
+                "-o",
+                "<reportFile>",
+                "<envVars>"
+            ],
+            "types": [
+                "jmeter/test"
```
Comment (vsukhin) [inline charts/testkube-api/executors.json:159]: jmeterd/test

File: charts/testkube-api/executors.json [hunk 2]
```diff
@@ -134,6 +134,46 @@
...
+                "file-uri",
+                "git-file",
+                "git-dir",
+                "git"
+            ],
+            "features": [
+                "artifacts"
+            ],
+            "meta": {
+                "iconURI": "jmeter",
```
Comment (vsukhin) [inline charts/testkube-api/executors.json:172]: will we use same docs and icon?

**Review by vsukhin — APPROVED** [review]
a couple of questions

File: charts/testkube-api/executors.json (hunk 2 above)
Comment (ypoplavs) [reply charts/testkube-api/executors.json:172]: probably different, but this is not a priority atm

**Pull request merged**